demos/stm32f10x/starter/makefile

```diff
@@ -0,0 +1,15 @@
+# sjsu_dev2.mk holds the $(SJSU_DEV2_BASE) variable which holds the location of
```

Comment:
Can you rename the folder to something like `blue_pill_reg_level_led_blink`. Something description. Maybe thats too long, but something to indicate which board this is for.